components/raftstore/src/store/peer_storage.rs

```diff
@@ -976,26 +996,23 @@ where
             panic!("{} unexpected state: {:?}", self.tag, *snap_state);
         }
 
-        if *tried_cnt >= MAX_SNAP_TRY_CNT {
```

Comment:
why remove this

Reply:
Snapshot generating can be kept canceling if the snapshot generator pool is busy.